architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -12,41 +12,38 @@
 toc::[]
 
 ifdef::openshift-origin,openshift-enterprise[]
-*Note: this only applies if the ephemeral-storage tech preview in
-{product-title} 3.10 has been enabled.  This is disabled by default;
-to enable this feature, please see
-xref:../../install_config/configuring_ephemeral.adoc#install-config-configuring-ephemeral-storage[Configuring for
-Ephemeral Storage].*
+[NOTE]
+====
+This topic only applies if the ephemeral storage technology feature preview 
+{product-title} 3.10 has been enabled. This feature is disabled by default. To enable this feature, see
+xref:../../install_config/configuring_ephemeral.adoc#install-config-configuring-ephemeral-storage[configuring for ephemeral storage].
+====
 endif::openshift-origin,openshift-enterprise[]
 
 == Overview
```

Comment:
I'd move the note under this heading and make it more descriptive.

Reply:
Thanks. Can you let me know if this more descriptive before I commit the changes?

"This topic applies only if the ephemeral storage technology preview is enabled in {product-title} 3.10. This feature is disabled by default. If enabled, the {product-title} cluster uses ephemeral storage to store information that does not need to persist after the cluster is destroyed. To enable this feature, see..."

Reply:
That sounds good.
Will you replace "Overview" with something more descriptive, too?